.github/workflows/update_pytorch_labels.yml

```diff
@@ -11,17 +11,24 @@ concurrency:
 jobs:
   update-labels-in-S3:
     runs-on: ubuntu-22.04
+    permissions:
+      id-token: write
+      contents: read
     if: ${{ github.repository == 'pytorch/pytorch' }}
     steps:
       - name: Checkout PyTorch
         uses: pytorch/pytorch/.github/actions/checkout-pytorch@main
         with:
           fetch-depth: 1
           submodules: false
+
+      - name: Configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::308535385114:role/gha_workflow_s3_and_ecr_read_only
```

Comment:
It looks like we need another role here to grant access to `ossci-metrics`.  This role doesn't cover that bucket.